examples/XML/statuses/Blind.xml

```diff
@@ -0,0 +1,22 @@
+<Status Duration="30" Tick="1" Icon="3" Name="Blind">
```

Comment:
These should not be under examples

Reply:
Just a note, if the statuses are being kept, need to add xmlns="http://www.hybrasyl.com/XML/Status", otherwise server start generates "NullReferenceException" error.

Reply:
Yes, fixed.